examples/pytorch/ModelSignature/iris.py

```diff
@@ -0,0 +1,186 @@
+import argparse
+import mlflow.pytorch
+import pandas as pd
+import pytorch_lightning as pl
+import torch
+import torch.nn as nn
+from mlflow.models import infer_signature
+from pytorch_lightning.metrics.functional import accuracy
+from sklearn.datasets import load_iris
+from torch.nn import functional as F
+from torch.utils.data import SubsetRandomSampler
+from torch.utils.data import TensorDataset
+from torch.utils.data import random_split
+from torch.utils.data.dataloader import DataLoader
+
+input_df = None
+output_df = None
+
+
+class IRISDataModule(pl.LightningDataModule):
```

Comment:
We already have an iris example - https://github.com/mlflow/mlflow/blob/master/tests/pytorch/iris.py . Can you please re-use the example ?

Reply:
Addressed